python/py-gpstime/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-gpstime
+version             0.3.2
+
+platforms           darwin
+supported_archs     noarch
+license             GPL-3
+maintainers         gmail.com:stefan.countryman
+
+python.rootname     gpstime
+
+description         GPS-aware datetime module with leap-second fetching
+long_description    This package provides a gpstime package, including a \
+                    gpstime subclass of the built-in datetime class with the \
+                    addition of GPS access and conversion methods.  Leap \
+                    second data is provided by the ietf_leap_seconds module \
+                    that helps automatically maintain a local copy of the \
+                    IETF leap second list: \
+                    https://www.ietf.org/timezones/data/leap-seconds.list \
+                    A command-line GPS data conversion utility that uses the \
+                    gpstime module is also included.  It is a rough \
+                    work-alike to tconvert.
+                    
+homepage            https://git.ligo.org/cds/${python.rootname}
+
+master_sites        https://git.ligo.org/cds/${python.rootname}/-/archive/${version}/
+distname            ${python.rootname}-${version}
+extract.suffix      .tar.gz
+
+checksums           rmd160  d62957cd15570ce24e5e17d5fe6d587777698809 \
+                    sha256  b303807ba1e6a93f1ee1d47f36c014f2580348dd5045dd0cb28c1a59a0d700aa \
+                    size    23900
+
+python.versions         35 36
+
+python.default_version  36
```

Comment:
No need for this line, as this would be the default version to be installed.

Reply:
Thanks! Removed, will push a new commit with all fixes after fixing livecheck (below).